core/templates/dev/head/pages/topics_and_skills_dashboard/SkillsListDirective.js

```diff
@@ -15,14 +15,12 @@
 /**
  * @fileoverview Controller for the skills list viewer.
  */
-oppia.constant(
-  'MERGE_SKILL_URL', '/merge_skill');
-
 oppia.directive('skillsList', [
-  '$http', 'AlertsService', 'UrlInterpolationService',
-  'MERGE_SKILL_URL',
+  '$http', 'AlertsService', 'TopicsAndSkillsDashboardBackendApiService',
```

Comment:
Not sure whether we should add this dependency to the controller or to the directive?

Reply:
Yeah, it was to the controller. Thanks!
Done.